app/components/structural/project-list-map.js

```diff
@@ -72,25 +71,44 @@ export const projectPolygonsHoverLayer = {
   },
 };
 
-// this component renders a map and receives a tiles URL
-// for rendering dynamic tiles. It's responsible for some styling
-// and is very much on the domain-problem side of the spectrum
-// it also yields out some contextual components that are invoked
-// from the consuming end. it also threads down an action to determine
-// that tile mode.
-
-// tile mode is what determines whether we see polygon or centroids.
+/**
+  * This component renders a map and receives a tiles URL
+  * for rendering dynamic tiles. It's responsible for some styling
+  * and is very much on the domain-problem side of the spectrum
```

Comment:
i am a n00b; can you explain what you mean by "domain-problem" and "consuming end"?

Reply:
Hmm I mean that it's something structural as opposed to generic? The "consuming end" means where the component is invoked. 

Will change language for clarity1